linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_kmod/ansible/ubuntu2004.yml

```diff
@@ -0,0 +1,51 @@
+# platform = multi_platform_ubuntu
```

Comment:
you can drop this, we are using the templated ansible, unless you see something wrong with it

Reply:
Templated ansible produces:
```
-a always,exit -S all -F path=/bin/kmod -F perm=x -F auid>=1000 -F auid!=-1 -F key=privileged
```
When DISA benchmark is expecting:
```
-w /bin/kmod -p x -k modules 
```

Reply:
then take a look at rule `audit_rules_privileged_commands_insmod`, there are some macros to help you there without needing to write the whole ansible or bash

Reply:
Thank you @dodys , will be taking a look at this soon

Reply:
Fixed ansible remediation to use macros. Thanks for the feedback!